include/openssl/opensslv.h

```diff
@@ -40,7 +40,9 @@ extern "C" {
  *  major minor fix final patch/beta)
  */
 # define OPENSSL_VERSION_NUMBER  0x10102000L
-# define OPENSSL_VERSION_TEXT    "OpenSSL 1.1.2-dev  xx XXX xxxx"
+# define OPENSSL_VERSION_NUMBER_TEXT "1.1.2-dev"
+# define OPENSSL_VERSION_TEXT "OpenSSL " OPENSSL_VERSION_NUMBER_TEXT \
+                              "  xx XXX xxxx"
```

Comment:
Also, this new define needs to be documented, see:

```
msp@msppc:~/openssl/openssl$ rgrep OPENSSL_VERSION doc/
doc/man3/OPENSSL_VERSION_NUMBER.pod:5:OPENSSL_VERSION_NUMBER, OpenSSL_version,
doc/man3/OPENSSL_VERSION_NUMBER.pod:11: #define OPENSSL_VERSION_NUMBER 0xnnnnnnnnnL
doc/man3/OPENSSL_VERSION_NUMBER.pod:20:OPENSSL_VERSION_NUMBER is a numeric release version identifier:
doc/man3/OPENSSL_VERSION_NUMBER.pod:54:=item OPENSSL_VERSION
```

Reply:
And while you're at it, would you mind documenting `OPENSSL_VERSION_TEXT`, too?  :wink:

Reply:
>Also, this new define needs to be documented, see:

Ah thanks, I'll add it. 


> And while you're at it, would you mind documenting `OPENSSL_VERSION_TEXT`, too?

No problem!